scripts/ssl-common.sh

```diff
@@ -7,13 +7,15 @@ if [ -z "${easyrsa_bin-}" ] || [ ! -x "${easyrsa_bin}" ]; then
     if [ -z "${easyrsa_bin}" ]; then
         easyrsa_dir="$(mktemp -dt easyrsa.XXXXXXXX)"
         easyrsa_url="https://github.com/OpenVPN/easy-rsa/releases/download/v3.0.5/EasyRSA-nix-3.0.5.tgz"
+        echo "  - Downloading EasyRSA..."
         (cd "${easyrsa_dir}"; curl -sL "${easyrsa_url}" | tar xz --strip-components=1)
         easyrsa_bin="${easyrsa_dir}/easyrsa"
         # shellcheck disable=SC2064
         trap "rm -rf \"${easyrsa_dir}\"" EXIT
     fi
     export EASYRSA_BATCH=1
     export EASYRSA_KEY_SIZE=4096
+    echo "  - Downloading EasyRSA...Done"
```

Comment:
This will print "Downloading ..." unconditionally every time the script is run, which doesn't provide any useful info. Even in the case where you really start downloading (i.e. you enter the `if` branch above) you don't need to print anything when it's done.